templates/docker-entrypoint.j2

```diff
@@ -1,7 +1,22 @@
-#!/bin/bash -e
+#!/bin/bash
 
+set -euo pipefail
+
+# Get a list of {{ beat }} subcommands (like "{{ beat }} setup") by consulting
+# the help output.
+subcommands=$({{ beat }} help | awk 'BEGIN {RS=""; FS="\n"} /Available Commands:/ {print}' | awk '/^\s+/ {print $1}')
```

Comment:
Love this part! perhaps you can avoid this call for the common case by moving all this to the `else` block after normal flags case?